content/automate/ManageIQ/System/Event/EmsEvent/Ansible_Tower.class/_missing.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Wondering if the source reference should be changed to be a mixed-case value here and return `AnsibleTower` which would be inline with what is expected.

The original source value was `VC` and the events added after all followed the same uppercase pattern.

Examples:
[app/models/manageiq/providers/openstack/cloud_manager/event_parser.rb#L10](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/openstack/cloud_manager/event_parser.rb#L10)
[app/models/manageiq/providers/google/cloud_manager/event_parser.rb#L14](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/google/cloud_manager/event_parser.rb#L14)
[app/models/manageiq/providers/storage_manager/cinder_manager/event_parser.rb#L12](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/storage_manager/cinder_manager/event_parser.rb#L12)

@lfu Do you know if this would adversely effect anything?

Any thoughts?  @Fryguy @blomquisg